.gitignore

```diff
@@ -63,3 +63,6 @@ admin.macaroon
 
 # built files
 dist/
+
+.idea
+package-lock.json
```

Comment:
`package-lock.json` should be included in the repo by the specs https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md